src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
+    }
+
+    public void stopWork() {
```

Comment:
The `stopWork` method should be annotated with `@Override` to clearly indicate that it is overriding a method from the superclass `Machine`.